editor/src/messages/portfolio/document/node_graph/node_graph_message_handler.rs

```diff
@@ -360,8 +361,10 @@ impl NodeGraphMessageHandler {
 		if reconnect {
 			// Check whether the being-deleted node's first (primary) input is a node
 			if let Some(node) = network.nodes.get(&deleting_node_id) {
-				if matches!(node.inputs.first(), Some(NodeInput::Node { .. })) {
-					first_input_node = Some(node.inputs[0].clone());
+				// Reconnect to the node below when deleting a layer node.
+				let index = if node.name == "Layer" { 7 } else { 0 };
```

Comment:
It's unclear to me what these magic numbers mean.

Reply:
It is the input index, allowing "the node below when deleting a layer node" (from the comment above).